src/views/v2/emails/template-parts/body/event/title.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Event Tickets Emails: Main template > Body > Event > Title.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/emails/template-parts/body/event/title.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
+ *
+ * @see tribe_get_event() For the format of the event object.
+ */
+
+if ( empty( $event ) && empty( $event->title ) ) {
+	return;
+}
+?>
+<tr>
+	<td style="padding:0;">
+		<h3 class="tec-tickets__email-table-content-event-title">
```

Comment:
This is for an email - why a class and not inline styles? Why just here?

Reply:
Not entirely sure. I am not going to modify this piece as I dont understand what the intended behavior might have been, since it's an email thing.

Reply:
@codingmusician can you chime in on this please?

Reply:
I don't see any new styles, so...

Reply:
Good question. I do know that when viewing the email in "Preview" mode, the admin styles tend to take over and add some default styling, but re-styling them can be accomplished using classes rather than being inline. You just have to use `td.class_name_here` to be specific.

Reply:
See this comment: https://github.com/the-events-calendar/the-events-calendar/pull/4196#issuecomment-1521901854